### PR TITLE
Added pkill neutron-ns-metadata-proxy per ctracey

### PR DESCRIPTION
**Pull request by nealbrown**

### CODE REVIEW

File: roles/security_errata/tasks/CVE-2015-7547.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  tags: CVE-2015-7547
+
+- name: kill neutron-ns-metadata-proxy
+  shell: pkill -f /usr/local/bin/neutron-ns-metadata-proxy
+  ignore_errors: true
+  when: glibc_minor_ver.stdout|int < 7
+  tags: CVE-2015-7547
+
+- name: restart neutron-dhcp-agent
+  service: name=neutron-dhcp-agent state=restarted
```
Comment (j2sol) [inline roles/security_errata/tasks/CVE-2015-7547.yml]: This needs to be neutron-l3-agent instead.

**Comment by nealbrown** [comment]
retest

**Comment by j2sol** [comment]
retest

**Comment by nealbrown** [comment]
retest

**Comment by nealbrown** [comment]
retest

**Comment by nealbrown** [comment]
retest

**Comment by nealbrown** [comment]
retest

**Pull request merged**